Kajabi/pine
# [FEATURE] - Update the Figma Icon export process

# Description

The process has been updated to handle some additional required features. Upon completing the download process, we needed to add the following
- identify which files have been created, modified, and deleted.
-  Build a changelog file. 
- Commit the changes

## How to test 
In order to kick off the process to test the functionality, you can follow the steps below.

1.  Confirm that you have NODE 18 installed. If you are running nvm, you can run one of two commands
  - `nvm install 18` <- if you don't have it installed
  - `nvm use` <- this will read the .nvmrc file and set the nod version to `18` 
2. Get the necessary Figma secrets. Check 1Password - Figma API Token
3. Update the `figma-icon-config.json` file to include
  - "figmaAccessToken": "[TOKEN obtained in step 1]"
  - "figmaFileId": "gr2OhPSlvtsYIZQCL5G2ey"
4. In your terminal window
  - `yarn setup` <- This just makes sure you have all the latest dependencies
  - `yarn nx run icons:update` <- Runs the process from start to finish.

If all goes well, there should be a commit with the message `ci(nightly): figma icons update`


## Fixes #(issue)
DSS-317

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Please describe the tests that you've added and run to verify your changes.
Provide instructions, so we can reproduce.
Please also list any relevant details for your test configuration.


**Test Configuration**:

- sage versions:
- os:
- browsers:
- screen readers:
- misc:

# Checklist:
